frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -185,6 +185,10 @@ pods:
         {{/TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
       restore-snapshot:
         goal: FINISHED
+        configs:
```

Comment:
There are some other tasks that also don't explicitly specify this file in the `configs`. Is this file implicitly used in any of these tasks?